### PR TITLE
VEGA-4074 -  Update LPA form sections on case subtype selection

### PR DESCRIPTION
**Pull request by MikeRJustice**
Fixes [VEGA-4074](https://opgtransform.atlassian.net/browse/VEGA-4074)

- Add Javascript module to toggle the LPA form sections depending on the users choice of case subtype (HW/PFA)
- Add unit test

[VEGA-4074]: https://opgtransform.atlassian.net/browse/VEGA-4074?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by almo2k — DISMISSED** [review]
Nice and clean overall. 

Unless I'm missing something I think there's a gap in the test — all three tests check the radio, then dispatch a change event, But None of them test what happens when a radio is already checked before lpaFormSubtype() runs (i.e. edit mode, where the subtype's pre-selected on page load and no change event fires). 

If you agree with my understanding then it might be worth adding a test for that case to cover it all off?

File: web/assets/lpa-form-subtype.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ export default function lpaFormSubtype(scope) {
 
     hwSection.hidden = value !== "hw";
     pfaSection.hidden = value !== "pfa";
+
+    console.log(pfaSection.hidden);
```
Comment (almo2k) [inline web/assets/lpa-form-subtype.js]: Is this a leftover debug?

**Review by almo2k — DISMISSED** [review]

**Review by almo2k — APPROVED** [review]
